documentation/src/docs/deployment/run-local.md

```diff
@@ -9,31 +9,31 @@ You may obtain a copy of the License at
 -->
 # Running Pravega
 
-Running in local machine allows us to get started using Pravega very quickly. Most of the options uses the standalone mode which is suitable for most of the development and testing purposes.
+
+Running in local machine allows us to get started using Pravega very quickly. Running Pravega in Standalone mode is suitable for development and testing Pravega applications.
 
 The prerequisites for running in local machine is described below.
 
 ## Standalone Mode
 
 ### From Source
 
-The Pravega source code needs to be checked out to download the dependencies:
+The Pravega source code needs to be checked out:
```

Comment:
edited

Reply:
One more thing here. Can we use active voice here (and everywhere else in this document): "Download the source code to your computer."?

Reply:
Accepted suggestion